packages/ingest-mongodb-public/.env.example

```diff
@@ -1,26 +1,16 @@
-PORT=3000
+# Chatbot App
+MONGODB_CONNECTION_URI=<connection_uri>
+MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
 OPENAI_ENDPOINT=https://<resource_name>.openai.azure.com/
 OPENAI_API_KEY=<api_key>
 OPENAI_EMBEDDING_DEPLOYMENT=<deployment_name>
-OPENAI_EMBEDDING_MODEL=text-embedding-ada-002
-OPENAI_EMBEDDING_MODEL_VERSION=2023-03-15-preview
-OPENAI_CHAT_COMPLETION_DEPLOYMENT=<deployment_name>
-OPENAI_CHAT_COMPLETION_MODEL_VERSION=<model_version>
-MONGODB_CONNECTION_URI=<connection_uri>
-MONGODB_DATABASE_NAME=<docs-chatbot-ENV>
-VECTOR_SEARCH_INDEX_NAME=<index_name>
-NODE_ENV=development
-# Connection string for DevCenter
-DEVCENTER_CONNECTION_URI="mongodb+srv://<username>:<password>@devhub-cluster.sewho.mongodb.net/?retryWrites=true&w=majority"
-# For connecting to MongoDB University Data API
-UNIVERSITY_DATA_API_KEY=abc123
-# For use with GitHubDataSource. The token only needs read access to the repo.
-GITHUB_ACCESS_TOKEN=ghp_TOKENTOKENTOKEN
 
```

Comment:
```
# Connection string for DevCenter
DEVCENTER_CONNECTION_URI="mongodb+srv://<username>:<password>@devhub-cluster.sewho.mongodb.net/?retryWrites=true&w=majority"
# For connecting to MongoDB University Data API
```

i think this is needed

Reply:
Are you sure? All of the tests pass without it + we make no reference to it in `ingest-mongodb-public` (although we probably should move _all_ of the devcenter/mdbu stuff into our mongodb ingest package instead of core ingest)

Reply:
> we probably should move all of the devcenter/mdbu stuff into our mongodb ingest package instead of core ingest

yeah, and the snooty stuff too

Reply:
yeah, it seems we import it directly from the env in the constructor (bad!) https://github.com/mongodb/chatbot/blob/main/packages/mongodb-rag-ingest/src/sources/DevCenterDataSource.ts#L38

so for now, we should keep this env var here. and maybe we could have a follow up ticket to pass this as a property to the `makeDevCenterDataSource()` constructor.